src/app/pipeline-detail/pipeline-detail.component.ts

```diff
@@ -69,12 +69,12 @@ export class PipelineDetailComponent implements OnInit {
 
   genReadmeURL(stage, job) {
     // example === https://github.com/mayadata-io/gui-automation/blob/master/testcases/gui-dashboard-alerts-check.md
-    if (stage == 'DIRECTOR-GUI-CHECK') {
+    if (stage == 'DIRECTOR-GUI-CHECK' ||stage == 'DIRECTOR-GUI') {
       let host = "https://github.com/mayadata-io/gui-automation/tree/master/testcases"
-      return host + '/' + job + '.md'
+      return host + '/' + job 
```

Comment:
Will the readme open if we remove `.md`?

Reply:
@amitbhatt818  created a issue for the path changes instead of readme  https://github.com/mayadata-io/oep-e2e/issues/596

Reply:
@bhaskarhc we need to put the test plan link instead of github readme. But this change will not take us to the testplan.

Reply:
Got it now . Thank yoh @harshshekhar15